lib/debezium/converters/timestamp.go

```diff
@@ -10,7 +10,7 @@ import (
 type Timestamp struct{}
```

Comment:
* Timestamp
* MicroTimestamp
* NanoTimestamp

Do not contain any timezone information.